account_move_locking/README.rst

```diff
@@ -0,0 +1,52 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Account move locking
+====================
+
+This module add the ability to lock move for modification 
```

Comment:
This module add the ability to lock and prevent any modification of given financial moves 

qqch comme ça non ?